docs/writing-schemas/scalars.md

```diff
@@ -25,39 +25,38 @@ extended scalar types provided by `graphql-java`.
 ## ID
 
 GraphQL supports a the scalar type ID, a unique identifier that is not intended to be human readable. ID's are
-serialized as a String. To mark given field as an ID field you have to annotate it with `@GraphQLID` annotation.
+serialized as a String. In order to expose a GraphQL `ID` field you have use `com.expediagroup.graphql.types.ID` class that wraps underlying String value.
```

Comment:
"To expose a GraphQL `ID` field, you must use the `com.expediagroup.graphql.types.ID` class, which wraps the underlying `String` value.